README.md

```diff
@@ -1,10 +1,17 @@
 Infusion Documentation
 ======================
 
+## What is infusion and how do I get it? ##
+
+See the infusion website at:
+
+<http://fluidproject.org/infusion.html>
+
+
 ## Browse the Infusion Documentation on GitHub
 
+You can [browse the Infusion Documentation as static user friendly pages](http://docs.fluidproject.org/infusion/development/index.html), or if you prefer, you can [browse the Infusion Documentation directly on GitHub](src/documents).
```

Comment:
@kaspermarkus this is great. Could you revise the new link to be "http://docs.fluidproject.org/infusion/" ? Right now, this redirects to /development/index.html anyway, and after discussion with @jobara, we feel this would be more appropriate.